### ISSUE TITLE
Linux. key ~ "Display current knowledge" doesn't open information

### ISSUE THREAD

**Issue by s88100** [draconisPW/PWMAngband#339]
After starting linux server, key ~ "Display current knowledge" prints "Display object, rune, artifact" ..., but doesn't open information. Connected to windows server, it works.

**Comment by draconisPW** [comment]
Hope my Linux VM still works to test this...

**Comment by s88100** [comment]
and doesn't work "@" key - display connected players

**Comment by s88100** [comment]
/src/server/knowledge-ui.c
do_cmd_knowledge_objects()
do_cmd_knowledge_runes()
do_cmd_knowledge_artifacts()  // etc.
```
    /* Temporary file */
    fff = file_temp(file_name, sizeof(file_name));
    if (!fff) return;
```

/src/common/z-file.c :L515
```
ang_file *file_temp(char *fname, size_t len)
{
#ifdef WINDOWS
    .....
#else
    /* TODO: UNIX VERSION! */
    **return NULL;**
#endif
}
```

**Comment by draconisPW** [comment]
This should be fixed by PR #406.
